const catchAsyncError = require("./catchAsyncError");

exports.isTokenExpired = catchAsyncError( async (req, res, next) => {
  
})